docs/tech.md

```diff
@@ -142,11 +143,31 @@ There is also evidence of code attempting to do the following:
 * Steal crypto wallets
 
 Jars are heuristically detected as Minecraft mods or plugins as follows:
-* Forge (dev/neko/e/e/e/A): The malware attempts to locate the `@Mod` annotation, which is required in every mod
-* Bukkit (dev/neko/e/e/e/C): The malware checks if a class extends Bukkit's `JavaPlugin` class
-* Fabric/Quilt (dev/neko/e/e/e/i): The malware checks if a class implements `ModInitializer`
-* Bungee (dev/neko/e/e/e/l): The malware checks if a class extends Bungee's `Plugin` class
-* Vanilla (dev/neko/e/e/e/c): The malware checks if the main client class `net.minecraft.client.main.Main` exists
+* Forge (`dev/neko/e/e/e/A`): The malware attempts to locate the `@Mod` annotation, which is required in every mod
+* Bukkit (`dev/neko/e/e/e/C`): The malware checks if a class extends Bukkit's `JavaPlugin` class
+* Fabric/Quilt (`dev/neko/e/e/e/i`): The malware checks if a class implements `ModInitializer`
```

Comment:
how is this check performed? quilt has a different package for its ModInitializer

Reply:
The bullet points for the unobf version further below explain more in detail

Reply:
Yeah this wont affect quilt then, unless someone was relying on qfapi and ignored the depreciation notice